test/katex-spec.js

```diff
@@ -2348,20 +2353,21 @@ describe("An extensible arrow parser", function() {
 
 describe("An extensible arrow builder", function() {
     it("should not fail", function() {
-        expect("\\xrightarrow{x}").toBuild();
-        expect("\\xrightarrow{x}^2").toBuild();
-        expect("\\xrightarrow{x}_2").toBuild();
-        expect("\\xrightarrow{x}_2^2").toBuild();
-        expect("\\xrightarrow[under]{over}").toBuild();
+        expect`\xrightarrow{x}`toBuild();
+        expect`\xrightarrow{x}^2`toBuild();
+        expect`\xrightarrow{x}_2`toBuild();
+        expect`\xrightarrow{x}_2^2`toBuild();
+        expect`\xrightarrow[under]{over}`toBuild();
+
```

Comment:
CI reveals that `.`s are missing in these examples. You can use `yarn` and then `yarn test:jest` locally to reproduce CI testing.